model/postgres/postgres_server.rb

```diff
@@ -101,7 +101,8 @@ def configure_hash
         }
       },
       identity: resource.identity,
-      hosts: "#{resource.representative_server.vm.private_ipv4} #{resource.identity}"
+      hosts: "#{resource.representative_server.vm.private_ipv4} #{resource.identity}",
+      pgbouncer_instances: vm.vcpus / 2
```

Comment:
Maybe we should set a fixed maximum number. 30 pgbouncers for standard-60 seems excessive. I'd go with

```
(vm.vcpus / 4).ceil.clamp(1, 8)
```

BTW, I don't have any scientific thought behind this formula. It just feels alright. If you have a better idea, that would work as well.

Reply:
Yeah it makes sense not to create too many of them for the larger instances. I was mostly going by our tests for `standard-8`, where we created 4 instances which was close enough to the original numbers, thinking to keep the factor as 2.

Reply:
Yeah, but our test was designed specifically to hit pgbouncer's weak spot and make it work as much as possible. I'd expect real world workloads to be less severe. That said I don't have strong arguments for 4 as a factor anyway and more importantly this is not difficult to change if we turned out to be wrong. So keeping the factor as 2 is also fine.

Let's not use more than 8 pgbouncer processes though.